benchmarks/run.py

```diff
@@ -109,6 +109,11 @@
             ("examples.matmul_split_k", "matmul_split_k_tritonbench"),
         ],
     ),
+    "fused_linear_jsd": (
+        "tritonbench.operators.fused_linear_jsd.operator",
+        "examples.fused_linear_jsd",
+        "fused_linear_jsd_fwd_tritonbench",
+    ),
```

Comment:
would you like to show result of running the benchmark with `--metrics accuracy`, to see if all accuracy checks pass?